ReactNativeClient/lib/models/Setting.js

```diff
@@ -333,7 +333,36 @@ class Setting extends BaseModel {
 			'encryption.passwordCache': { value: {}, type: Setting.TYPE_OBJECT, public: false, secure: true },
 			'style.zoom': { value: 100, type: Setting.TYPE_INT, public: true, appTypes: ['desktop'], section: 'appearance', label: () => _('Global zoom percentage'), minimum: 50, maximum: 500, step: 10 },
 			'style.editor.fontSize': { value: 13, type: Setting.TYPE_INT, public: true, appTypes: ['desktop'], section: 'appearance', label: () => _('Editor font size'), minimum: 4, maximum: 50, step: 1 },
-			'style.editor.fontFamily': { value: '', type: Setting.TYPE_STRING, public: true, appTypes: ['desktop'], section: 'appearance', label: () => _('Editor font family'), description: () => _('This must be *monospace* font or it will not work properly. If the font is incorrect or empty, it will default to a generic monospace font.') },
+			'style.editor.fontFamily': {
+				value: '',
+				type: Setting.TYPE_STRING,
+				public: true,
+				appTypes: ['desktop'],
+				section: 'appearance',
+				label: () => _('Editor font family'),
+				description: () =>
+					_('This must be *monospace* font or it will not work properly. If the font ' +
+					'is incorrect or empty, it will default to a generic monospace font.'),
+			},
+			editorFont: {
+				value: Setting.FONT_MENLO,
+				type: Setting.TYPE_STRING,
+				isEnum: true,
+				public: true,
+				label: () => _('Editor font'),
+				appTypes: ['mobile'],
+				section: 'appearance',
+				options: () => ({
+					/**
+					 * TODO: Check if these fonts break Android. If so, how to retrieve
```

Comment:
Unfortunately nothing happens on Android. Whatever font I select, it always looks the same and the font is not monospace.

Reply:
I suspect the two monospace fonts I picked are not available on Android then. Can you try changing "Menlo" to "monospace" or "Roboto Mono" to see what happens?

Reply:
Also this means it'll require having some way to handle the different cases available for Android vs iOS. I'm sure this is possible somehow, but from my quick poking around (e.g. of [this `platform` variable](https://github.com/devonzuegel/joplin/blob/50d510970b9b3cec93e8bcc5cf9a7ae5d1886b49/ReactNativeClient/lib/models/Setting.js#L24)) it wasn't imminently obvious how to fetch that. 

Before sinking lots of time into it I'm curious to hear what @laurent22 thinks. (No rush! 🙂)

Reply:
Nope, didn't help either.

Reply:
> No rush! 

I think Laurent's on vacation or took a short break. I haven't really got any feedback from him lately.